lib/vite.config.js

```diff
@@ -6,18 +6,21 @@ const name = "storyblok-nuxt";
 export default defineConfig(() => {
   return {
     build: {
+      // target: "esnext",
       lib: {
         entry: path.resolve(__dirname, "index.js"),
         name: "storyblokNuxt",
         fileName: (format) => (format === "es" ? `${name}.mjs` : `${name}.js`),
       },
       rollupOptions: {
         output: {
+          compact: true,
           globals: {
             vue: "Vue",
+            "@storyblok/vue-2": "@storyblok/vue-2",
           },
         },
-        external: ["vue"],
+        external: ["vue", "@storyblok/vue-2"],
```

Comment:
That means that the end user would need to install both, the `@storyblok/vue-2` and `@storyblok/nuxt-2` 

You would need to update the readme and add `@storyblok/vue-2`  as a peer dep